src/Audio/Properties/AssemblyInfo.cs

```diff
@@ -1,33 +1,8 @@
 ﻿using System.Reflection;
-using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-// Les informations générales relatives à un assembly dépendent de 
-// l'ensemble d'attributs suivant. Changez les valeurs de ces attributs pour modifier les informations
-// associées à un assembly.
 [assembly: AssemblyTitle("sfml-audio")]
 [assembly: AssemblyDescription("Audio module of the SFML library")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("SFML")]
-[assembly: AssemblyCopyright("Copyright © Laurent Gomila")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
 
-// L'affectation de la valeur false à ComVisible rend les types invisibles dans cet assembly 
-// aux composants COM. Si vous devez accéder à un type dans cet assembly à partir de 
-// COM, affectez la valeur true à l'attribut ComVisible sur ce type.
 [assembly: ComVisible(false)]
```

Comment:
Not a big issue, but while you are changing this, would you stuff mind to move the com visible attribute also and set it to true? (all others except audio are visible)